src/mlx_omni_server/chat/mlx/model_types.py

```
@@ -63,13 +81,150 @@ def load_mlx_model(
+                    model_id=model_id,
+                    adapter_path=adapter_path,
+                    draft_model_id=draft_model_id,
+                    model=wrapped_model,
+                    tokenizer=wrapped_tokenizer,
+                    chat_template=chat_template,
+                    is_vlm_model=True,
+                )
+                
+            except ImportError as e:
+                logger.error(f"mlx_vlm not available: {e}")
+                raise RuntimeError(f"Model {model_id} requires mlx_vlm but it failed to import: {e}")
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)